modules/creating-config-files-cluster-install-oci.adoc

```diff
@@ -137,22 +137,22 @@ bootArtifactsBaseURL: <server_URL> <4>
 <1> The cluster name that you specified in your DNS record.
 <2> The namespace of your cluster on {product-title}.
 <3> If you use IPv4 as the network IP address format, ensure that you set the `rendezvousIP` parameter to an IPv4 address that the VCN’s Classless Inter-Domain Routing (CIDR) method allocates on your network. Also ensure that at least one instance from the pool of instances that you booted with the ISO matches the IP address value you set for `rendezvousIP`.
-<4> The URL of the server where you want to upload the `rootfs` image.
+<4> The URL of the server where you want to upload the rootfs image.
 
 . Apply one of the following two updates to your `agent-config.yaml` configuration file:
 +
-* For a disconnected network:  After you run the command to generate a minimal ISO Image, the Agent-based installer saves the `rootfs` image into the `./<installation_directory>/boot-artifacts` directory on your local system. Use your preferred web server, such as any Hypertext Transfer Protocol daemon (`httpd`), to upload `rootfs` to the location stated in the `bootArtifactsBaseURL` parameter in the `agent-config.yaml` configuration file.
+* For a disconnected network:  After you run the command to generate a minimal ISO Image, the Agent-based installer saves the rootfs image into the `./<installation_directory>/boot-artifacts` directory on your local system. Use your preferred web server, such as any Hypertext Transfer Protocol daemon (`httpd`), to upload rootfs to the location stated in the `bootArtifactsBaseURL` parameter in the `agent-config.yaml` configuration file.
 +
-For example, if the `bootArtifactsBaseURL` parameter states \http://192.168.122.20, you would upload the generated `rootfs` image to this location, so that the Agent-based installer can access the image from \http://192.168.122.20/agent.x86_64-rootfs.img. After the Agent-based installer boots the minimal ISO for the external platform, the Agent-based Installer downloads the `rootfs` image from the \http://192.168.122.20/agent.x86_64-rootfs.img location into the system memory.
+For example, if the `bootArtifactsBaseURL` parameter states \http://192.168.122.20, you would upload the generated rootfs image to this location, so that the Agent-based installer can access the image from \http://192.168.122.20/agent.x86_64-rootfs.img. After the Agent-based installer boots the minimal ISO for the external platform, the Agent-based Installer downloads the rootfs image from the \http://192.168.122.20/agent.x86_64-rootfs.img location into the system memory.
```

Comment:
```suggestion
For example, if the `bootArtifactsBaseURL` parameter states `\http://192.168.122.20`, you would upload the generated `rootfs` image to this location, so that the Agent-based installer can access the image from `\http://192.168.122.20/agent.x86_64-rootfs.img`. After the Agent-based installer boots the minimal ISO for the external platform, the Agent-based Installer downloads the `rootfs` image from the `\http://192.168.122.20/agent.x86_64-rootfs.img` location into the system memory.
```

Reply:
I think you may need some backticks here?

Reply:
AH, OK. I'll try this agin and hopefully the backslash does not render.
![Screenshot from 2024-04-18 09-24-59](https://github.com/openshift/openshift-docs/assets/57954076/52670f71-507c-4598-97ce-bfa553b2f374)